docs/common/cursor_c_lua.ldoc

```diff
@@ -0,0 +1,80 @@
+-- <div class='flex-list'>
```

Comment:
This line shouldn't have the initial `--` because when the build tool copies the file content, there is already a comment marker in the destination file (`--@DOC_cursor_c_lua_COMMON@`). Btw this "error" should be visible in the documentation after you generate it.

Reply:
Fixed.